packages/orbit-components/src/Box/README.md

```diff
@@ -171,15 +171,26 @@ If you want to define `spacingObject`, you can define specific directions like t
 
 ### spacingToken
 
-| spacingToken |
-| :----------- |
-| `"none"`     |
-| `"XXXSmall"` |
-| `"XXSmall"`  |
-| `"XSmall"`   |
-| `"small"`    |
-| `"medium"`   |
-| `"large"`    |
-| `"XLarge"`   |
-| `"XXLarge"`  |
-| `"XXXLarge"` |
+| spacingToken                                |
+| :------------------------------------------ |
+| `"none"`                                    |
+| `"XXXSmall"` - **deprecated (use `"50"`)**  |
+| `"XXSmall"` - **deprecated (use `"100"`)**  |
+| `"XSmall"` - **deprecated (use `"200"`)**   |
+| `"small"` - **deprecated (use `"300"`)**    |
+| `"medium"` - **deprecated (use `"400"`)**   |
+| `"large"` - **deprecated (use `"600"`)**    |
+| `"XLarge"` - **deprecated (use `"800"`)**   |
+| `"XXLarge"` - **deprecated (use `"1000"`)** |
```

Comment:
Same here, XXXLarge missing. Even if it is deprecated, it can still be used, right?

Reply:
It can, but we don't want it to be used, as it is about to be not supported anymore. So we still accept it, but don't document it nor encourage its usage anymore. The ones with direct replacement are documented so one knows what to migrate to. WDYT?